docs/day-2-operations/Backups.adoc

```diff
@@ -144,8 +144,105 @@ Backup the dump.rb file from redis
 oc cp $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r):/var/lib/redis/data/dump.rdb ./system-redis-dump.rdb
 ----
 
+=== `Secrets`
```

Comment:
I don't see the following secrets included:

```
backend-listener
backend-redis
system-memcache
```

Can we add them to the backup list?

Additionally, in case of S3 being used there's a secret for the credentials. Should we include it in the backup? If so, we should figure out a way to identify the secret name being used. Probably querying some DeploymentConfig environment variable secretRef to identify it.

Finally, when using the official templates/operator, there's the hardcoded "threescale-registry-auth" secret. Should we include that secret too?

Take into account that something similar will have to be done in the 3scale-operator in the future

Reply:
@miguelsorianod 

* `backend-listener` is not required. Its content can be reproduced by the template call. 
* `backend-redis` is not required. Its content is only useful for "External databases" deployment mode. Its content can be reproduced by the template call. This backup&restore procedure only is valid for the "default" profile. In the [jira issue](https://issues.redhat.com/browse/THREESCALE-4683), I already pointed it out. The contents

```
One more issue about the backup&restore procedure. 3scale has several deployment options. The backup&restore procedure is assuming the basic 3scale installation. The documentation lack backup restore procedure for deployment options such as:

* 3scale with postgresql
* 3scale using S3 as system storage
* 3scale HA mode (external databases)
```
* `system-memcache`:  is not required. Its content can be reproduced by the template call. 

* The `threescale-registry-auth` should not be backuped up. It contains credentials of the customer that the customer is already keeping it safe.

Reply:
What is the criteria/considered "can be reproduced by the template"? the procedure creates the template using the contents of the ConfigMaps/Secrets

Reply:
Means that the `oc new-app --file ...` command run by the customer according to the manual, will create those secrets/configmaps with the exact same value. No need to backup/restore them. In other words, their content is not the source of truth.

Reply:
I think this comment is dependent on whether we allow ConfigMaps/Server pre-creation. Under an operator based deployment we do. Under templates I'm not aware of the official position about this. It would not surprise some users do it (even if our official position is that we don't)

Reply:
Ok, I will add missing secrets and configmaps